.ado/templates/fluentui-apple-publish-nuget-job.yml

```diff
@@ -1,7 +1,7 @@
 jobs:
 - job: fluentui_apple_publish_nuget
   pool:
-    vmImage: 'macos-10.15'
+    vmImage: 'macos-11.0'
```

Comment:
curious if this works

Reply:
The OS selection seems to be working as per the log below.
But it looks like #486 changes made the work flow stop using our script to select the correct Xcode version.

![Screen Shot 2021-05-10 at 3 18 57 PM](https://user-images.githubusercontent.com/68076145/117731481-0e24a400-b1a3-11eb-8d09-c7e109846318.png)